### PR TITLE
Fix README.md codes

### PR DESCRIPTION
**Pull request by kokorinosoba**
Hi, thank you for developing the adjust-xamairin-SDK.

I found the not-working codes for iOS in the README.md when I tried to install and built my app.
I also found the android codes that were not consistent with the rest. So I fixed it.

Could you please check it out?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1158,9 +1158,9 @@ Adjust.AppDidLaunch(config);
 **For Android app:**
 
 ```cs
-AdjustConfig adjustConfig = AdjustConfig(this, appToken, environment);
-adjustConfig.SetAppSecret(secretId, info1, info2, info3, info4);
-Adjust.OnCreate(adjustConfig);
+AdjustConfig config = AdjustConfig(this, appToken, environment);
+config.SetAppSecret(secretId, info1, info2, info3, info4);
+Adjust.OnCreate(config);
```
Comment (kokorinosoba) [inline README.md:1163]: Not consistent with the rest of android codes
adjustConfig → config

**Comment by uerceg** [comment]
Hi @kokorinosoba 

Thanks for the contribution! Makes perfect sense. 👍

File: README.md (hunk 1 above)
Comment (uerceg) [reply README.md:1163]: I agree. Snippet should still be functional, but nice to have iOS and Android ones synced in naming. 👌

**Comment by kokorinosoba** [comment]
Thanks for the review! If everything is fine, could you please merge?

**Comment by uerceg** [comment]
Replaced with https://github.com/adjust/xamarin_sdk/pull/35. Thanks for your contribution one more time! 🍺

**Comment by kokorinosoba** [comment]
Thanks for your kindness 😊
If I find anything else, I'll make a pull request.

**Pull request closed without merging**